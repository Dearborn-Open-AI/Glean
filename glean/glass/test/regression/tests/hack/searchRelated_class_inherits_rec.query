action: searchRelated
args: {"symbol":"test/php/SourceInterface","request":{"relatedBy":1,"relation":2,"recursive":true,"detailedResults":true}}
