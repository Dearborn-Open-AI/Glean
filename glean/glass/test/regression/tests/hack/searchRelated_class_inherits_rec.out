[
    "@generated",
    {
        "edges": [
            {
                "child": "test/php/RefClass",
                "parent": "test/php/SourceInterface"
            }
        ],
        "symbolDetails": {
            "test/php/RefClass": {
                "annotations": [],
                "comments": [],
                "kind": 7,
                "language": 3,
                "location": {
                    "filepath": "www/RefClass.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 10,
                        "lineEnd": 32
                    },
                    "repository": "test"
                },
                "name": {
                    "container": "",
                    "localName": "RefClass"
                },
                "repo_hash": "testhash",
                "signature": "class RefClass",
                "sym": "test/php/RefClass",
                "sym_location": {
                    "filepath": "www/RefClass.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 10,
                        "lineEnd": 32
                    },
                    "repository": "test"
                },
                "sym_other_locations": []
            },
            "test/php/SourceInterface": {
                "annotations": [],
                "comments": [],
                "kind": 13,
                "language": 3,
                "location": {
                    "filepath": "www/SourceInterface.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 10,
                        "lineEnd": 16
                    },
                    "repository": "test"
                },
                "name": {
                    "container": "",
                    "localName": "SourceInterface"
                },
                "repo_hash": "testhash",
                "signature": "interface SourceInterface",
                "sym": "test/php/SourceInterface",
                "sym_location": {
                    "filepath": "www/SourceInterface.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 10,
                        "lineEnd": 16
                    },
                    "repository": "test"
                },
                "sym_other_locations": []
            }
        }
    }
]